groups/sig-scheduling/groups.yaml

```diff
@@ -79,6 +79,16 @@ groups:
       - acondor@google.com
       - handbomusic@gmail.com
 
+  - email-id: k8s-infra-staging-wasm-scheduler@kubernetes.io
+    name: k8s-infra-staging-wasm-scheduler
+    description: |-
+      ACL for staging kube-scheduler-wasm-extension images.
+    settings:
+      ReconcileMembers: "true"
+    members:
+      - handbomusic@gmail.com
+      - kerthcet@gmail.com
```

Comment:
@kerthcet I added you to this list so that it won't be only me.
Let me know if you want to have a different email.

Reply:
Yes, the right one. Thanks.
/lgtm